api/v1/testkube.yaml

```diff
@@ -3474,6 +3474,10 @@ components:
         token:
           type: string
           description: git auth token for private repositories
+        isBitbucketServerToken:
```

Comment:
let's call isHeaderToken, because it might be useful for other git providetrs

Reply:
Done